builder/index_full.html

```diff
@@ -1118,28 +1128,271 @@
                                     </div>
                                 </div>
                             </div>
+                            <div class="col mb-1">
+                                <div class="row row-cols-1 row-cols-1 text-center scaled-font dark-5 rounded dark-shadow">
+                                    <div class="col fw-bold dark-4 rounded-top">
+                                        Active boosts
+                                    </div>
+                                    <div class="col mb-1">
+                                        <div class="col">
+                                            <button class="button-boost m-1 border-0 text-white dark-8u dark-shadow-sm" id="vanish-boost" onclick="update_boosts('vanish-boost')">
+                                                Vanish (+80%)
+                                            </button>
+                                            <button class="button-boost m-1 border-0 text-white dark-8u dark-shadow-sm" id="warscream-boost" onclick="update_boosts('warscream-boost')">
```

Comment:
Looks like duplicated code? Can you remove the old set

Reply:
cleaned up duplicate codes